drivers/tty/serdev/pibridge_trace.h

```diff
@@ -220,19 +371,39 @@ DECLARE_EVENT_CLASS(pibridge_timeout_class,
 	)
 );
 
+/*
+ * pibridge_receive_timeout
+ *
+ * Info: timeout occurred while waiting for data reception.
+ * Time: after the timeout occurred.
+ */
 DEFINE_EVENT(pibridge_timeout_class, pibridge_receive_timeout,
 	TP_PROTO(unsigned int received, unsigned int expected,
 		 unsigned int timeout),
 	TP_ARGS(received, expected, timeout)
 );
 
+/*
+ * pibridge_discard_timeout
+ *
+ * Info: a timeout occurred while receiving a larger than expected packet and
+ *       the excess data was going to be discarded.
```

Comment:
Hm, I find this difficult to parse. How about replacing "and" with a comma and inserting "of which" after "the excess data", i.e.:

"a timeout occurred while receiving a larger than expected packet, the excess data of which was going to be discarded."